custom-targets/vertex-ai/advanced_quickstart/ADVANCED_QUICKSTART.md

```diff
@@ -0,0 +1,369 @@
+       --format="value(projectNumber)")-compute@developer.gserviceaccount.com \
+       --role="roles/clouddeploy.jobRunner"
+   ```
+2. `roles/clouddeploy.viewer` - required to access Cloud Deploy resources
+
+   ```shell
+   gcloud projects add-iam-policy-binding $PROJECT_ID \
+       --member=serviceAccount:$(gcloud projects describe $PROJECT_ID \
+       --format="value(projectNumber)")-compute@developer.gserviceaccount.com \
+       --role="roles/clouddeploy.viewer"
+   ```
+3. `roles/aiplatform.user` - required to access the models and deploy endpoints in the custom target
+
+   ```shell
+   gcloud projects add-iam-policy-binding $PROJECT_ID \
+       --member=serviceAccount:$(gcloud projects describe $PROJECT_ID \
+       --format="value(projectNumber)")-compute@developer.gserviceaccount.com \
+       --role="roles/aiplatform.user"
+   ```
+
+4. Build and Register a Custom Target Type for Vertex AI
+
+From within the `quickstart` directory, run this command to build the Vertex AI model deployer image and
+install the custom target resources:
+
+```shell
+../build_and_register.sh -p $PROJECT_ID -r $REGION
+```
+For information about the `build_and_register.sh` script, see the [README](../README.md#build)
+
+
+## 4. Import a model into Model Registry
+
+We will upload a pre-existing model to Model Registry before deploying it with Cloud Deploy.
+The `gcloud ai models upload` command requires a cloud storage path to the trained model, and a
+docker image to run the model in.
+For this quickstart, we will be using a pre-trained model from Vertex AI's cloud samples bucket.
+For the docker image, we use a Cloud AI optimized version of tensorflow: [tf_opt](https://cloud.google.com/vertex-ai/docs/predictions/optimized-tensorflow-runtime).
+
+   ```shell
+   gcloud ai models upload \
+       --artifact-uri gs://cloud-samples-data/vertex-ai/model-deployment/models/boston/model \
+       --display-name=test-model \
+       --container-image-uri=us-docker.pkg.dev/vertex-ai-restricted/prediction/tf_opt-cpu.nightly:latest \
+       --project=$PROJECT_ID \
+       --region=$REGION \
+       --model-id=test_model
+   ```
+
+If this is your first time using Vertex AI in this project, this operation will
+take 10 minutes or so.
+
+## 5. Create Vertex AI Endpoints
+
+Create the Vertex AI endpoints using the following commands:
+
+   ```shell
+   export DEV_ENDPOINT_ID="dev"
+   gcloud ai endpoints create --display-name $DEV_ENDPOINT_ID --endpoint-id $DEV_ENDPOINT_ID --region $REGION --project $PROJECT_ID
+   ```
+
+   ```shell
+   export PROD_ENDPOINT_ID="prod"
+   gcloud ai endpoints create --display-name $PROD_ENDPOINT_ID --endpoint-id $PROD_ENDPOINT_ID --region $REGION --project $PROJECT_ID
+   ```
+
+If this is your first time using Vertex AI in this project, this operation will
+take 5 minutes or so.
+
+The endpoint ID will be used to refer to the endpoint, rather than the display name.
```

Comment:
Do you mean "We'll use the endpoint ID rather than the display name to refer to the endpoint"?